packages/celotool/src/cmds/account/verify.ts

```diff
@@ -102,61 +81,53 @@ async function verifyCmd(argv: VerifyArgv) {
 }
 
 export async function printCurrentCompletedAttestations(
-  attestations: AttestationsType,
+  attestations: AttestationsWrapper,
   phoneNumber: string,
   account: string
 ) {
-  const attestationStat = await attestations.methods
-    .getAttestationStats(PhoneNumberUtils.getPhoneHash(phoneNumber), account)
-    .call()
+  const attestationStat = await attestations.getAttestationStat(phoneNumber, account)
 
   console.info(
     `Phone Number: ${phoneNumber} has completed ${
-      attestationStat[0]
-    } attestations out of a total of ${attestationStat[1]}`
+      attestationStat.completed
+    } attestations out of a total of ${attestationStat.total}`
   )
 }
 
+async function send(obj: Promise<CeloTransactionObject<any>>) {
```

Comment:
we should have a method on CeloTransactionObject that does this. `tellMeWhenItsDone(): Promise<Result>`
where result is something like:
```
Result :: NotMined | MinedAndFailed txHash receipt | Success txHash receipt
```
(i know you like the notation :P)

Or you can go bananas, and have something like `Promise<Try<receipt>>` and two types of errors :P

Reply:
I see you still don't like my functional style :P

I added a simple wrapper to `CeloTransactionObject`. Would leave an abstraction to a future PR

Reply:
I like it! but i find this could be useful for many other users... so having it hidden here is not great.

It's just that i don't consider `obj.function()` not functional, just because there's an obj prefix ;). And i don find that useful when you don't know the library... you have intellisense and can find "what you can do" with the txObj.... finding the `send` as topLevel function in some utility module is harder...

but we can leave this for another pr if you want